src/main/scala/mesosphere/marathon/ZookeeperConf.scala

```diff
@@ -16,16 +16,16 @@ trait ZookeeperConf extends ScallopConf {
   private val zkURLPattern = s"""^zk://(?:$userAndPass@)?($hostAndPort(?:,$hostAndPort)*)(/$zkNode(?:/$zkNode)*)$$""".r
 
   lazy val zooKeeperTimeout = opt[Long]("zk_timeout",
-    descr = "The timeout for ZooKeeper in milliseconds",
+    descr = "The timeout for Zookeeper in milliseconds.",
     default = Some(10 * 1000L)) //10 seconds
 
   lazy val zooKeeperSessionTimeout = opt[Long]("zk_session_timeout",
-    descr = "The timeout for zookeeper sessions in milliseconds",
+    descr = "The timeout for Zookeeper sessions in milliseconds",
     default = Some(10 * 1000L) //10 seconds
   )
 
   lazy val zooKeeperUrl = opt[String]("zk",
-    descr = "ZooKeeper URL for storing state. Format: zk://host1:port1,host2:port2,.../path",
+    descr = "Zookeeper URL for storing state. Format: zk://host1:port1,host2:port2,.../path",
```

Comment:
By me. I cared mostly for consistency but to speak for Gaston's variant, that's how ZooKeeper spells itself: https://zookeeper.apache.org/

"Zookeeper" is more in line with the usual English spelling.

You choose.